### PR TITLE
Update OWNERS_ALIASES to reflect new team structure

### PR DESCRIPTION
**Pull request by nikhita**
I'm not completely sure if the cluster-lifecycle and seed teams are reflected accurately through this change, happy to punt those two teams to another PR.
/hold

/cc @kron4eg @xmudrii 
/assign @alvaroaleman @xrstf @zreigz 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
     - kdomanski
     - moadqassem
-    - nikhita
     - thz
-    - themue
+    - kron4eg
+    - xmudrii
 
   team-seed:
-    - kron4eg
```
Comment (kron4eg) [inline OWNERS_ALIASES:12]: in order to help @xrstf with reviews I need to be in both teams

**Review by kron4eg — APPROVED** [review]
/lgtm

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-    - nikhita
     - thz
-    - themue
+    - kron4eg
+    - xmudrii
 
   team-seed:
     - kron4eg
     - xrstf
     - thetechnick
```
Comment (xrstf) [inline OWNERS_ALIASES]: Is Nico really part of the seed team?

**Review by kron4eg — COMMENTED** [review]
/lgtm

**Pull request merged**